packages/react-instantsearch-theme-algolia/styles/_Menu.scss

```diff
@@ -51,3 +54,32 @@
 
 .ais-Menu__itemLabelSelected{
 }
+
+$sffv-searchbox-config:(
```

Comment:
you probably need to rename this variable to make it unique so it does not impact the other searchbox config.